.travis.yml

```diff
@@ -25,7 +25,7 @@ deploy:
     skip_cleanup: true
     email: npm@khouri.ca
     api_key:
-      secure: SlI8U43IpEctxgzP1H+GhxdxPE2VNNuSfA85YZdnlQ/gMQ2hifZQYP1q9k3z2H4RRdo+ooEkmcsXE0a/s6XGrTIO98p4zI2LapOsEgedNsYwBWmLSwq0r2LCXDc5btvr043NdPFkhhfhNcmCtYFuWrAMLjg4ypttBcbMdMPj+Jw=
+      secure: EuVF1Ji3RIO6JF91ww4GIx5qUY2xiaPUfYA92uvqiGwJam7taL/v3OLy6Y5khjnXxFKgfkjd+x2tQ9n2hS56t9Ky7ud8Zw439an/PqEWGE3olTv4n7ZdVaAHzcIydlNL5qmezdlTDM+tmggD7gWWsjatYc9QY4fSnGUtdRZ4HzM=
```

Comment:
You need to use UI to add secured tokens, do not expose tokens in source code

https://blog.travis-ci.com/2018-04-11-how_to_publish_node_js_packages_with_travis_ci_and_packagecloud/

Reply:
@nkbt this line is using [Travis's encryption](https://docs.travis-ci.com/user/encryption-keys/) to hide the API keys. Based on the Travis documentation, only the CI server will be able to decrypt this key 🔑 so I think we're in good shape here

Reply:
I've seen that in docs. But does it mean that this key is encrypted per account? So if you are using different account to encrypt it - it will not work?

Either way for better security I would avoid keeping any tokens in github. Encrypted or not. Especially if there is an easier way with encrypted ENV vars.

Reply:
Based on the docs, the encryption key is per-repo. Since this method is documented in Travis's docs, I don't see a security issue with it. For UI vs travis.yml, I'd prefer to keep the key in travis.yml, since then there is only one single point of config for Travis (rather than travis.yml + some changes in the UI)